src/components/tasks/card/index.tsx

```diff
@@ -258,13 +262,171 @@ const Card: FC<Props> = ({
+                                    width: `${content.percentCompleted}%`,
+                                }}
+                            ></div>
+                        </div>
+                        <span>{content.percentCompleted}% </span>
+                    </div>
+                </div>
+                <div className={classNames.dateInfo}>
+                    <div>
+                        <span className={classNames.cardSpecialFont}>
+                            Estimated completion{' '}
+                        </span>
+                        {renderDate(fromNowEndsOn, shouldEdit)}
+                    </div>
+                    <span
+                        className={classNames.cardSpecialFont}
+                        contentEditable={shouldEdit}
+                        onKeyPress={(e) => handleChange(e, 'startedOn')}
+                        role="button"
+                        tabIndex={0}
+                    >
+                        {cardDetails.status == 'Available'
+                            ? 'Not started '
+                            : `Started on ${fromNowStartedOn}`}
+                    </span>
+                </div>
+
+                <div className={classNames.cardItems}>
+                    <div className={classNames.contributor}>
+                        <span className={classNames.cardSpecialFont}>
+                            Assigned to{' '}
+                        </span>
+                        <span className={classNames.contributorImage}>
+                            <Image
+                                src={assigneeProfilePic}
+                                alt="Assignee profile picture"
+                                onError={contributorImageOnError}
+                                width={30}
+                                height={30}
+                            />
+                        </span>
+                        <span
+                            className={classNames.cardStrongFont}
+                            contentEditable={shouldEdit}
+                            onKeyPress={(e) => handleChange(e, 'assignee')}
+                            role="button"
+                            tabIndex={0}
+                        >
+                            {cardDetails.assignee}
+                        </span>
+                    </div>
+                    <div
+                        className={`${classNames.taskTagLevelWrapper} ${
+                            shouldEdit && classNames.editMode
+                        }`}
+                    >
+                        <div className={classNames.taskTagLevelContainer}>
+                            {taskTagLevel?.map((item) => (
+                                <span
+                                    key={item.tagId}
+                                    className={classNames.taskTagLevel}
+                                >
+                                    {item.tagName}{' '}
```

Comment:
Again, don't add static empty string. Use CSS for spacing

Reply:
done